react/react-training/src/constants/index.js

```diff
@@ -6,6 +6,12 @@ export const LINES = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2,
 
 export const OFFLINE_MESSAGE = 'Sorry. We are offline. Please come back in a few minutes.';
 
+export const TARGE_USERDATA = 'userData';
+
+export const TARGE_USERISLOGGED = 'userIsLogged';
+
+export const TARGET_USERLOGIN = 'userLogin';
```

Comment:
it isn't necessary to declare this const here if you aren't going to use it in many files. This constants are used only in one file. So, I'd move this three constants to that files